Content.Server/Corvax/StationGoal/StationGoalPrototype.cs

```diff
@@ -17,5 +17,11 @@ public sealed class StationGoalPrototype : IPrototype
         [DataField]
         public int? MaxPlayers;
 
+        /// <summary>
+        /// purpose may require certain items. These items will appear near the captain's fax machine at the start of the round
+        /// TODO: They should be spun up at the tradepost instead of at the fax machine, but I'm too lazy to do that right now. Maybe in the future.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Goal may require certain items to complete. These items will appear near the receving fax machine at the start of the round.
        /// TODO: They should be spun up at the tradepost instead of at the fax machine, but I'm too lazy to do that right now. Maybe in the future.
        /// </summary>
```